examples/Demo/DocGenerator/CodeCommentsGenerator.cs

```diff
@@ -79,6 +79,8 @@ private static string CleanupParamName(string value)
         value = regex.Replace(value, "");
         regex = new("[P,T,M,F]:FluentUI\\.Demo\\.Shared\\.");
         value = regex.Replace(value, "");
+        regex = new("\\([^()]*\\)");
```

Comment:
If ChatGPT give me the correct answer: "The goal of this regular expression (\([^()]*\)) is to match any sequence of characters that is enclosed within parentheses, without any nested parentheses inside."

Is it possible to be more precise to avoid replacing s’ (abc) ’ written by ourselves in the documentation?

Reply:
I did some further testing after drafting and found a few more issues. This fix does not address overloaded methods. For example for `ToastService.ShowToast`, the fix is only returning the description for the first.

@dvoituron  I'm working on another fix which will reverse the last commit.

Reply:
@dvoituron 

It appears that this may be the best approach. That is, to correct the `key` being generated in the `CodeCommentGenerator`.  The intention is the remove parameter description from the "method name" in the entry created in the generated `CodeComments` class.

``` text
["ToastService.ShowToast``2(Microsoft.FluentUI.AspNetCore.Components.ToastParameters{``1})"] = "Shows the toast with the component type", 
```
to
``` text
["ToastService.ShowToast``2"] = "Shows the toast with the component type",
```

Good catch on the regular expression 😆, It's been a while since I've used regular expressions so I used ChatGPT to generate the initial expression. I'm using `"\\(.*\\)$"` now.  It will remove anything between the first opening parenthes and last closing parentheses including the parentheses on any string ending with a closing parentheses. 🤞